Sources/protoc-gen-swift/EnumGenerator.swift

```diff
@@ -113,8 +128,8 @@ class EnumGenerator {
     } else {
       p.print("\(visibility)static let _protobuf_nameMap: SwiftProtobuf._NameMap = [\n")
       p.indent()
-      for c in enumCases {
-        p.print("\(c.number): .same(proto: \"\(c.protoName)\"),\n")
+      for c in enumCases where !c.isAlias {
```

Comment:
For this and the two loops below, does it make sense to use firstCases and sort based on the numbers?